omod/src/main/resources/config.xml

```diff
@@ -111,6 +111,29 @@
 		<defaultValue>false</defaultValue>
 		<description>Whether email should be sent for tele-consultation</description>
 	</globalProperty>
+	<globalProperty>
+		<property>bahmni.appointment.adhocTeleConsultation.patientEmailNotificationSubject</property>
+		<defaultValue>AdhocTeleconsultation started</defaultValue>

```

Comment:
Same comment. No need for the word "adhoc".

Reply:
Updated